.rubocop.yml

```diff
@@ -10,7 +10,7 @@ Layout/LineLength:
 Metrics/MethodLength:
   Max: 50
 Metrics/AbcSize:
-  Max: 100
+  Max: 150

```

Comment:
- [ ] [REQUIRED] Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))